src/components/_global/BalChart/BalChart.vue

```diff
@@ -191,16 +197,31 @@ const chartConfig = computed(() => ({
       ? tailwind.theme.colors.gray['900']
       : tailwind.theme.colors.white,
     formatter: params => {
+      let processedParams = params;
+
+      // if veBAL, reverse the order of the params to meet the design requirements
+      if (props.isVeBAL) {
+        processedParams = [...params.reverse()];
+        // filter equal values in same date
+        processedParams = processedParams.filter((param, index) => {
+          if (index === 0) return true;
+          return param.value[1] !== processedParams[index - 1].value[1];
+        });
+      }
```

Comment:
This component shouldn't really have any context of the actual chart usage, like veBAL. If in some cases like vebal you need to reverse the params the prop should probably be `reverseParams`

Reply:
I did it to match with design requirements. otherwise without reverse here, chart will not look ok